### PR TITLE
refactor(accordion-item): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

**Comment by paulcpederson** [comment]
screener is being really weird, it's highlighting tabs, which we didn't change? Also it is not highlighting the actual change I just made which is fixing accordion header height when there is no subtitle:

| before | after |
| --- | --- |
| Screen Shot 2021-03-23 at 10 36 49 AM | Screen Shot 2021-03-23 at 10 36 59 AM |

**Comment by eriklharper** [comment]
> screener is being really weird, it's highlighting tabs, which we didn't change? Also it is not highlighting the actual change I just made which is fixing accordion header height when there is no subtitle:
> 
> before	after
> Screen Shot 2021-03-23 at 10 36 49 AM	Screen Shot 2021-03-23 at 10 36 59 AM

The height of each row and the entire container does appear different in the after shot.  The after shot shows the text appears more centered in the row.  Is that not intended?

**Comment by paulcpederson** [comment]
@eriklharper no, that's the intentional change. screener doesn't think they're different, but they clearly are.

**Comment by macandcheese** [comment]
giphy-downsized

File: src/components/calcite-accordion-item/calcite-accordion-item.scss [hunk 1]
```diff
@@ -1,92 +1,88 @@
 // host-content() is being used to support IE11 styling in this component.
```
Comment (jrowlingson) [inline src/components/calcite-accordion-item/calcite-accordion-item.scss:1]: Can we remove these IE hacks?

**Comment by eriklharper** [comment]
Weird... I wonder if our pixel diffing threshold needs to be adjusted.  Can you include the link that shows the problem screenshot?  I'd like to pass that onto Loyal to see what he thinks.

File: src/components/calcite-accordion-item/calcite-accordion-item.scss [hunk 2]
```diff
@@ -157,63 +152,61 @@
...
+  @apply flex-col flex-grow my-0 py-0;
   margin-right: auto;
-  flex-direction: column;
-  flex: 1;
   text-align: initial;
 }
 
 :host([dir="rtl"]) .accordion-item-header-text {
+  @apply mr-0;
   margin-left: auto;
```
Comment (jrowlingson) [inline src/components/calcite-accordion-item/calcite-accordion-item.scss:162]: ml-auto

File: src/components/calcite-accordion-item/calcite-accordion-item.scss [hunk 3]
```diff
@@ -157,63 +152,61 @@
 
 // accordion item title
 :host .accordion-item-header-text {
+  @apply flex-col flex-grow my-0 py-0;
   margin-right: auto;
```
Comment (jrowlingson) [inline src/components/calcite-accordion-item/calcite-accordion-item.scss:156]: mr-auto

File: src/components/calcite-accordion-item/calcite-accordion-item.scss [hunk 4]
```diff
@@ -157,63 +152,61 @@
...
+  @apply text-color-2 font-medium;
 }
 :host .accordion-item-subtitle {
-  color: var(--calcite-ui-text-3);
+  @apply text-color-3 mt-1 leading-snug;
 }
 :host([dir="rtl"]) .accordion-item-title {
-  margin-right: 0;
+  @apply mr-0;
   margin-left: auto;
```
Comment (jrowlingson) [inline src/components/calcite-accordion-item/calcite-accordion-item.scss:178]: ml-auto et al

**Comment by julio8a** [comment]
merged this into accordion

**Pull request closed without merging**